tests/plugins/button/manual/label.html

```diff
@@ -0,0 +1,27 @@
+<span id="results"></span>
+<div id="editor"></div>
+
+<script>
+	var results = CKEDITOR.document.getById( 'results' ),
+		editor = CKEDITOR.replace( 'editor', {
```

Comment:
In such test editor's language should be forced, as in my editor there is no "Set language" button, but "Ustaw język" one ;)